router.js

```diff
@@ -49,6 +56,14 @@ router.get("/", async ctx => {
           map_id: "the id of the map, i.e. floor, wing, etc. See /workspaces.",
         },
       },
+      "/roombookings": {
+        description: "Returns non-departmental bookings",
+        parameters: {
+          roomid: "Room id for the room to fetch bookings for",
```

Comment:
Don't you need a `siteid` as well as a `roomid`? Or maybe I'm being dumb :)

Reply:
I thought I didn't because `siteid` was optional. But now I've just realised that `roomid`s are probably not unique, and so I probably need a `siteid` as well. #willfix